sections/1-introduction.tex

```diff
@@ -11,14 +11,14 @@ \section{Introduction}
 Finally, the solar community benefitted from functionality developed by the astronomy community (specifically, the IDL Astronomy User's Library).
 Since then, significant functionality has been developed by the solar community and freely distributed as the SolarSoftWare library \citep{Freeland:1998we}.
 
-Officially founded in March of 2014, the goal of the \sunpyproj is to provide the core functionality needed for solar data analysis in \python\footnote{\url{https://www.python.org/}}, a high-level interpreted  programming language, and facilitate a new transition to bring significant and new benefits to the solar community.
+Officially founded in March 2014, the goal of the \sunpyproj is to provide the core functionality needed for solar data analysis in \python\footnote{\url{https://www.python.org/}}, a high-level interpreted programming language, and facilitate a new transition to bring significant and new benefits to the solar community.
 The \sunpyproj develops and maintains a community-led, free, and open-source\footnote{\url{https://opensource.org/osd}} core \python package (\sunpypkg), supports an ecosystem of affiliated packages (see \autoref{sec:affil_package}) consistent with best practices \citep{Wilson:2014cka}, and engages with the community through mailing lists, chat rooms, tutorials, summer programs, and mentorship.
-The \sunpyproj shares these goals with the Astropy Project\footnote{\url{https://www.astropy.org}}, which develops the \astropypkg core package \citep{astropy2018} for the astrophysics community.
+The \sunpyproj has similar goals to the Astropy project\footnote{\url{https://www.astropy.org}}, which develops the \astropypkg core package \citep{astropy2018} for the astrophysics community.
```

Comment:
It can't say "shares these goals" with Astropy since the first goal is developing `sunpy`.